phonebook-api/src/main/java/com/telran/phonebookapi/model/CountryCode.java

```diff
@@ -0,0 +1,36 @@
+package com.telran.phonebookapi.model;
+
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@NoArgsConstructor
+@Getter
+public class CountryCode {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
make the code as id, remove @GeneratedValue

Reply:
GeneratedValue was removed, int code is using as "Id".